### DISCUSSION TITLE
Help with input

### DISCUSSION THREAD

**Discussion by Afifette** [Afifette/mdrlol#2] [Q&A] [answered]
Hey i thinks it's here i need to post
How to do a input ?

**Comment by W3bParserV2** [comment, marked as answer]
### I recommend you to learn the basics of the python before posting.

Just write :
```python
input("What is your name ?\nEnter your name :")
```
If you want to retrieve your input value, simply write some thing like that :
```python
name = input("What is your name ?\nEnter your name :")
print("Hello ", name)
#Or
print("Hello " + name)
```
**Here you are** 👍

**Reply by Afifette** [reply]
THXXXX

**Comment by Afifette** [comment]
THANKSKSSSS
